README.md

```diff
@@ -240,7 +240,9 @@ A sample configuration file with all options is available [here](https://github.
 * `no-eval` disallows `eval` function invocations.
 * `no-inferrable-types` disallows explicit type declarations for variables or parameters initialized to a number, string, or boolean.
 * `no-internal-module` disallows internal `module` (use `namespace` instead).
-* `no-invalid-this` disallows using the `this` keyword outside of classes.
+* `no-invalid-this` disallows using the `this` keyword in scopes where they are often misinterpreted
+    * `outside-of-class` disallows using the `this` keyword outside of classes.
```

Comment:
Can we keep this behavior on by default? For backwards-compat reasons, we don't want to have to make users add a new option to their rule to get the same behavior as before